lib/JsonStreamQueryTransformer.ts

```diff
@@ -0,0 +1,137 @@
+import {Transform} from "readable-stream";
+import {JsonEvent} from "json-event-parser";
+
+export type JsonValue = string | number | boolean | null | JsonValue[] | {[key: string]: JsonValue};
```

Comment:
Can/should this be exported by json-event-parser.js?

Reply:
It might. I have not put it there because this type won't be used anywhere in `json-event-parser.js` current API. But if we add query layer, then it might make sense.

Reply:
Done: https://github.com/comunica/json-event-parser.js/pull/1